src/common/utility.ts

```diff
@@ -448,3 +449,16 @@ export const handleRedirect = (residence, current_client_country) => {
 export const isLocalhost = () => !!(isBrowser() && process.env.NODE_ENV === 'development')
 
 export const isTestlink = () => !!(isBrowser() && window.location.hostname.includes('binary.sx'))
+
+export const isShowBranding = () => {
+    const host_name = isBrowser() && window.location.hostname
+    const domain_name = getDomainName()
+    return branding_domains.some((item) => item === host_name || item === domain_name)
+}
+
+export const getDomainName = () => {
+    const hostname = isBrowser() && window.location.hostname
+    return (
+        hostname && hostname.substring(hostname.lastIndexOf('.', hostname.lastIndexOf('.') - 1) + 1)
```

Comment:
The `window.location.hostname` will always return the domain name. Any reason for this string transformation using substring?

Reply:
yes needed as in some test links,we have the domian link as deriv-com-git-fork-aswathy-deriv-derivhidebranding.binary.sx' where we need only the domain name as binary..so we are using substring to get only the value binary

Reply:
You can use `^(?:https?:\/\/)?(?:[^@\/\n]+@)?(?:www\.)?([^:\/\n]+)` regex to capture the domain by matching the `location.href`. I think this is more reliable than checking for dots 🙏🏼

Reply:
Or you can check that string exist by this way:
`hostname.includes('deriv-com-git-fork')`
And show brand if exist

Reply:
i did the same @NikitK-deriv